core/src/avm2.rs

```diff
@@ -154,17 +153,13 @@ impl<'gc> Avm2<'gc> {
     /// Dispatch an event on an object.
     ///
     /// The `bool` parameter reads true if the event was cancelled.
-    pub fn dispatch_event(
+    pub fn dispatch_event_object(
         context: &mut UpdateContext<'_, 'gc, '_>,
-        event: Event<'gc>,
+        event_object: Object<'gc>,
```

Comment:
Bikeshedding: seems like renaming the fn is a little wordy to me, I prefer the shorter name, and it'd be obvious upon seeing the signature that it takes an Object.